composer.json

```diff
@@ -13,7 +13,7 @@
         "php": "^7.3",
         "laminas/laminas-component-installer": "^1.0 || ^2.1",
         "laminas/laminas-development-mode": "^3.2",
-        "laminas/laminas-skeleton-installer": "^0.1.7 || ^1.0",
+        "laminas/laminas-skeleton-installer": "^0.1.7 || ^0.2 ||  ^1.0",
```

Comment:
Please drop  0.1.7 from dependencies, We need only latest version here,

Reply:
done. but I think changing a major version of a constraint requires a minor version change, not just a patch release...